lua/gluatest/stubs/stubMaker.lua

```diff
@@ -18,7 +18,6 @@ return function()
             callHistory = {},
             restored = false,
             Restore = function( self )
-                if not original then return end
```

Comment:
If you stub a function onto a table that didn't already have a function at that key, calling `:Restore()` wouldn't remove it.

I think this makes the process technically cleaner?